Frameworks/CoreText/DWriteWrapper_CoreText.mm

```diff
@@ -204,7 +204,7 @@ bool _CloneDWriteGlyphRun(_In_ DWRITE_GLYPH_RUN const* src, _Out_ DWRITE_GLYPH_R
     for (size_t i = 0; i < [subString length]; i += attributeRange.length) {
         NSDictionary* attribs = [static_cast<NSAttributedString*>(string) attributesAtIndex:i + range.location
                                                                       longestEffectiveRange:&attributeRange
-                                                                                    inRange:{ i, [subString length] }];
+                                                                                    inRange:{ i + range.location, [subString length] }];
```

Comment:
I'll do this the next time I add an attribute we handle here or on any cleanup pass I make.